Menus/UndoMenu/UndoMenu.cs

```diff
@@ -120,14 +120,18 @@ public void ProcessInput(ActionMapInput input, ConsumeControlDelegate consumeCon
             var undoRedoPerformed = false;
             if (navigateX < -k_UndoRedoThreshold && (m_TrackpadController || m_PrevNavigateX > -k_UndoRedoThreshold))
             {
+#if UNITY_EDITOR

```

Comment:
Probably should do a utils version of undo/redo so we can avoid these ifdefs or ideally switch in playmode functionality

Reply:
The latest hackweek project of a C# undo system might work.

Reply:
@mtschoen-unity - you may want to look into this, too --^

Reply:
Yes. I will definitely be taking a look at the hack week project for undo/redo. I've tasked it out for now--out of scope for this PR.